### PR TITLE
Locations map

### PR DESCRIPTION
**Pull request by janphilipp-winkler**

### CODE REVIEW

File: components/LegacyAnimation/LegacyAnimation.js [hunk 1]
```diff
@@ -2,12 +2,13 @@ import * as Styled from "./LegacyAnimation.styled";
...
-      </Styled.BlobsContainer>
-    </Styled.Container>
+    <>
+      <Styled.Container color={color} $opacity={opacity}>
+        <Styled.BlobsContainer>
+          <Styled.Blob $duration={"2s"} color={color} />
+          <Styled.Blob $duration={"6s"} color={color} />
+        </Styled.BlobsContainer>
+      </Styled.Container>
+    </>
```
Comment (onemanwenttomow) [inline components/LegacyAnimation/LegacyAnimation.js:12]: We don't need the React Fragment here...

```suggestion

      <Styled.Container color={color} $opacity={opacity}>
        <Styled.BlobsContainer>
          <Styled.Blob $duration={"2s"} color={color} />
          <Styled.Blob $duration={"6s"} color={color} />
        </Styled.BlobsContainer>
      </Styled.Container>

```

File: components/MapOfCircles/MapOfCircles.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import React, { useEffect, useState, useRef } from "react";
```
Comment (onemanwenttomow) [inline components/MapOfCircles/MapOfCircles.js:1]: We no longer need to import `React` into our files. 

```suggestion
import  { useEffect, useState, useRef } from "react";
```

File: components/MapOfCircles/MapOfCircles.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+                Math.min(screenSize.width, screenSize.height) *
+                (0.2 / Math.log(entry.count + 3)),
+              10
+            )}
+            name={entry.experience || entry.region}
+            color={entry.color}
+          />
+        ))}
+      </Styled.Grid>
+    </>
```
Comment (onemanwenttomow) [inline components/MapOfCircles/MapOfCircles.js:42]: We don't need the React Fragment here...

```suggestion
      <Styled.Grid ref={containerRef}>
        {data[0].map((entry, index) => (
          <Circle
            key={index}
            count={entry.count}
            circleSize={Math.max(
              Math.sqrt(entry.count) *
                Math.min(screenSize.width, screenSize.height) *
                (0.2 / Math.log(entry.count + 3)),
              10
            )}
            name={entry.experience || entry.region}
            color={entry.color}
          />
        ))}
      </Styled.Grid>

```

**Review by onemanwenttomow — CHANGES_REQUESTED** [review]
Nice job team   🎉

I added some suggestions. If you decide to implement the suggestion regarding the return value from `locationAnalyser` you will need to make changes in multiple places.

**Comment by janphilipp-winkler** [comment]
> Nice job team   🎉
> 
> I added some suggestions. If you decide to implement the suggestion regarding the return value from `locationAnalyser` you will need to make changes in multiple places.

thanks a lot @onemanwenttomow for the nice suggestions. I implemented them all!

**Pull request merged**